### DISCUSSION TITLE
Prompt after delegating

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#88] [General]
the prompt after delegation is missing information , see below : 
![Screenshot 2023-05-25 at 14 14 47](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/e6c26147-6a2e-4808-a07a-27b8561e48fd)

**Comment by TJ-BitCanna** [comment]
It feels like 2 pictures are overlapping here. 

Is this the confirmation after sending a tx or when you are confirming the amount to be delegated?

**Reply by dbwebapp** [reply]
This is a confirmation prompt after a delegation.

**Reply by TJ-BitCanna** [reply]
Check!

So it overlaps with this issue it seems?
https://github.com/BitCannaGlobal/WeedWallet/discussions/66

**Comment by TJ-BitCanna** [comment]
Solved :)
